### PR TITLE
doc: Misc fixups for Transparent Encryption GSG

### PR DESCRIPTION
**Pull request by christarazi**
Updates https://github.com/cilium/cilium/issues/11903

### CODE REVIEW

**Comment by christarazi** [comment]
test-docs-please

File: Documentation/gettingstarted/encryption.rst [hunk 1]
```diff
@@ -12,33 +12,39 @@ Transparent Encryption (stable/beta)
 
 This guide explains how to configure Cilium to use IPsec based transparent
 encryption using Kubernetes secrets to distribute the IPsec keys. After this
-configuration is complete all traffic between Cilium
-managed endpoints, as well as Cilium managed host traffic, will be encrypted
-using IPsec. This guide uses Kubernetes secrets to distribute keys. Alternatively,
-keys may be manually distributed but that is not shown here.
+configuration is complete all traffic between Cilium managed endpoints, as well
```
Comment (pchaigno) [inline Documentation/gettingstarted/encryption.rst]: Nit (compound adjective):
```suggestion
configuration is complete all traffic between Cilium-managed endpoints, as well
```

**Review by pchaigno — APPROVED** [review]
One nit below; :+1: otherwise!

(Finally, a native speaker who uses commas :tada: That really helps to quickly parse sentences, not being a native speaker.)

**Comment by christarazi** [comment]
Travis hit https://github.com/cilium/cilium/issues/12111

File: Documentation/gettingstarted/encryption.rst [hunk 2]
```diff
@@ -59,16 +59,32 @@ Deploy Cilium release via Helm:
...
+    helm install cilium |CHART_RELEASE| \\
+      --namespace cilium \
+      --set global.nodeinit.enabled=true \
+      --set nodeinit.reconfigureKubelet=true \
+      --set nodeinit.removeCbrBridge=true \
+      --set global.cni.binPath=/home/kubernetes/bin \
+      --set global.tunnel=vxlan \
+      --set global.encryption.enabled=true \
+      --set global.encryption.nodeEncryption=false
+
```
Comment (jrfastab) [inline Documentation/gettingstarted/encryption.rst:82]: Its probably preferred to run GKE with direct routing but having an example is nice. We can add a direct routing example later.

**Review by jrfastab — APPROVED** [review]

File: Documentation/gettingstarted/encryption.rst (hunk 2 above)
Comment (borkmann) [reply Documentation/gettingstarted/encryption.rst:82]: Could we still add one as follow-up, so we don't loose track of it? Thx

**Pull request merged**